source/roadmap.html.erb

```diff
@@ -17,66 +17,78 @@ title: Roadmap
+        <h3 class="govuk-heading-m">Corporate exemptions</h3>
         <p class="govuk-body">
-          Services can now use external telephone payment systems to collect card details and send them to GOV.UK&nbsp;Pay via
-          <%= link_to 'a new API', 'https://docs.payments.service.gov.uk/send_card_details_api', class: 'govuk-link' %>. This allows these payments to be processed and reported in
-          the same way as online payments through GOV.UK&nbsp;Pay.
+          We integrated with Worldpay’s corporate card exemption engine to meet card scheme requirements for a selection of services who have a significant portion of business payees.
         </p>
         <h2 class="govuk-heading-l">What we’re doing now</h2>
-        <h3 class="govuk-heading-m">Learning about open banking</h3>
-        <p class="govuk-body">We're looking at using open banking, which could make payments faster and cheaper while keeping them secure.</p>
+
+        <h3 class="govuk-heading-m">Updating the settings page in the admin tool</h3>
+        <p class="govuk-body">
+          We’re making changes to the settings pages in the GOV.UK Pay admin tool so they're clearer and simpler. This will make it easier for public sector teams to manage their services and switch easily between live and test modes.
+        </p>
         <h3 class="govuk-heading-m">Accepting new payment types</h3>
         <p class="govuk-body">
-          Related to the open banking work, we're updating our system to manage not just card payments but other ways to pay as well. This change is important because it means
-          paying users of government services will have more options, making GOV.UK Pay able to adapt to different needs. More payment options make it simpler for everyone to use
-          government services in the way that works best for them, and allows more public services to benefit from using us. We're looking at using open banking, which could make
-          payments faster and cheaper while keeping them secure.
+         We're updating Pay to manage not just card payments but other ways to pay as well. This change is important because it means paying users of public sector services will have more options in future, making GOV.UK Pay able to adapt to different needs. More payment options make it simpler for everyone to use public sector services in the way that works best for them, and allows more public services to benefit from using us. We plan to use pay by bank, which could make payments faster and cheaper while keeping them secure.
+        </p>
+        <h3 class="govuk-heading-m">Procuring new card and pay by bank payment suppliers</h3>
+        <p class="govuk-body">
+          We're completing a procurement exercise to get the best card and pay by bank suppliers. This will help to make sure service teams and paying users have the best value for money and experience when using GOV.UK Pay.
```

Comment:
```suggestion
          We're completing procurement to get the best card and pay by bank suppliers. This will make sure service teams and paying users have the best value for money and experience when using GOV.UK Pay.
```

Reply:
Not sure on the procurement vs. procurement exercise - so feel free to ignore that bit of the suggestion.